scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -91,25 +94,18 @@ public static BuildTargetResult RestoreTestAssetProjects(BuildTargetContext c)
                 "--verbosity", "verbose",
                 "--infer-runtimes",
                 "--fallbacksource", Dirs.TestPackages,
-                "--fallbacksource", Dirs.Corehost)
+                "--fallbacksource", Dirs.Corehost,
+                "--fallbacksource", Dirs.CorehostDummyPackages)
                 .WorkingDirectory(Path.Combine(c.BuildContext.BuildDirectory, "TestAssets", "TestProjects"))
                 .Execute()
                 .EnsureSuccessful();
 
-            // The 'ProjectWithTests' is a portable test app. Cannot call --infer-runtimes on it, since on win x64 machines,
```

Comment:
Nice!

Just a heads up, this is going to conflict with #2527, where @pranavkm added more tests to this `ProjectWithTests` directory. Last one in will have to merge ;).